slidey_js.js

```diff
@@ -0,0 +1,27 @@
+$(document).ready(function(){
+
+$("#background");
+var images = [
+  "http://i.imgur.com/WnkBqTE.gif",
+	"http://i.imgur.com/KSWrk.gif",
+	"http://i.imgur.com/Bc6pb58.gif",
+	"http://i.imgur.com/gqQChS5.gif"];
+
+
+var x = 0;
+
```

Comment:
i noticed you use the variable `x` here, could you think of a more semantic variable name? what does the `x` represent? Maybe we could use something like `current_image` or something to that effect.